x-pack/plugins/cases/public/components/create/description.test.tsx

```diff
@@ -31,7 +32,7 @@ describe('Description', () => {
   };
 
   beforeEach(() => {
-    jest.resetAllMocks();
+    jest.clearAllMocks();
```

Comment:
I want to make sure this change is intended, because `resetAllMocks` does a bit more than `clearAllmocks` does.

resetMock will remove mocked implementation, values, basically return it to a new state, clearMock will only remove the existing calls count and results but keep existing mocked values and implementations. Depending on the test files that could pollute tests depending on which sequence they are run.

Reply:
Hey! Yes, this is intended. The `resetAllMocks` will remove any mocks that may be defined in a `__mock__` folder. I want to keep the implementation that is provided by a `__mock__` folder. Specifically for the `useLensDraftComment` hook.